content/tutorial-ios/ios-01.md

```diff
@@ -0,0 +1,126 @@
+# Tutorial 01 - Getting Started
+
+Welcome to the 1st exercise in the **iOS** Track of this Apollo Client Tutorial! If you prefer React, React Native, Angular 2 or Vue.js over iOS, head over to the respective tutorial track.
+
+> Note: This tutorial assumes a basic familiarity with GraphQL and focusses on functionality of the [**Apollo iOS Client**](http://dev.apollodata.com/ios/). If you want to learn more about the used GraphQL concepts, visit [learngraphql](www.learngraphql.com).
+
+## Goal
+
+The **goal** of this exercise is to get your environment set up so you can use the **Apollo iOS Client** in Xcode. 
+
+By the end of this exercise you will have a running Xcode project in which you can use the **Apollo iOS Client**. The app will display a generic greeting in a table view, telling you that you don't have any Pokemons in your Pokedex yet.
+
+
+## Introduction
+
+Sign up with GitHub to receive your own `pokedex-apollo` project here:
```

Comment:
Please be consistent with your usage of `pokedex-ios`, `pokedex-apollo`, `apollo-pokedex`.